pkg/beholder/config.go

```diff
@@ -0,0 +1,58 @@
+package beholder
+
+import (
+	"time"
+
+	otelattr "go.opentelemetry.io/otel/attribute"
+)
+
+type Config struct {
+	InsecureConnection       bool
+	CACertFile               string
+	OtelExporterGRPCEndpoint string
+
+	PackageName string
+	// OTel Resource
+	ResourceAttributes map[string]string
```

Comment:
How about using the eventual otel type here? We already have values of this type to pass in from the existing config.
```suggestion
	ResourceAttributes []otelattr.KeyValue
```

Reply:
https://github.com/smartcontractkit/chainlink-common/pull/696/files#diff-8ae7d9439c8e585e8376ebac5e4f399dbf8309df188de9882a8925be6c208159R96

Reply:
@jmank88 addressed https://github.com/smartcontractkit/chainlink-common/pull/694/commits/b8f6c6f310051c056d18f9ba9dd20b98812996ac